guava-tests/pom.xml

```diff
@@ -121,6 +121,18 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>exec-maven-plugin</artifactId>
+        <version>3.1.0</version>
+        <configuration>
+          <classpathScope>test</classpathScope>
+          <mainClass>com.google.caliper.runner.CaliperMain</mainClass>
+          <arguments>
+            <argument>${benchmark}</argument>
+          </arguments>
+        </configuration>
+      </plugin>
```

Comment:
Commands for running benchmarks (to do in VM):
`./mvnw -B -P!standard-with-extra-repos exec:java -U -Dmaven.javadoc.skip=true -Dbenchmark=com.google.common.collect.ConcurrentHashMultisetBenchmark -f pom.xml -pl guava-tests`
`./mvnw -B -P!standard-with-extra-repos exec:java -U -Dmaven.javadoc.skip=true -Dbenchmark=com.google.common.concurrent.AbstractFutureFootprintTest -f pom.xml -pl guava-tests`